app/_hub/kong-inc/kafka-upstream/index.md

```diff
@@ -156,30 +182,28 @@ This plugin makes use of [lua-resty-kafka](https://github.com/doujiang24/lua-res
 
 When encoding request bodies, several things happen:
 
-* For requests with content-type header of `application/x-www-form-urlencoded`, `multipart/form-data`
-  or `application/json`, this plugin will pass the request body "raw" on the `body` attribute, but also try
-  to return a parse version of those arguments in `body_args`. If this parsing fails, an error message will be
-  returned and the message will not be sent.
-* If the `content-type` is not `text/plain`, `text/html`, `application/xml`, `text/xml` or `application/soap+xml`,
+* For requests with content-type header of `application/x-www-form-urlencoded`, `multipart/form-data`,
+  or `application/json`, this plugin will pass the raw request body on the `body` attribute, but also tries
+  to return a parsed version of those arguments in `body_args`. If this parsing fails, an error message is
+  returned and the message is not sent.
+* If the `content-type` is not `text/plain`, `text/html`, `application/xml`, `text/xml`, or `application/soap+xml`,
   then the body will be base64-encoded to ensure that the message can be sent as JSON. In such a case,
-  the message will have an extra attribute called `body_base64` set to `true`.
+  the message has an extra attribute called `body_base64` set to `true`.
 
 ## Known issues and limitations
 
 Known limitations:
 
-1. There is no support for TLS
-2. There is no support for Authentication
-3. There is no support for message compression
-4. The message format is not customizable
+1. There is no support for TLS.
+2. There is no support for Authentication.
+3. There is no support for message compression.
+4. The message format is not customizable.
 
 ## Quickstart
 
-The following guidelines assume that both `Kong` and `Kafka` have been installed on your local machine:
-
-1. Install `kong-plugin-kafka-upstream` as mentioned on the "Installation" section of this readme.
+The following guidelines assume that both `Kong` and `Kafka` have been installed on your local machine.
```

Comment:
If `Kong Enterprise` is installed on your local machine, then it already comes with `Kafka-Log` bundled with it.

Reply:
> If `Kong Enterprise` is installed on your local machine, then it already comes with `Kafka-Log` bundled with it.

Thanks for confirming. I know you meant kafka upstream, not log.